napari/_qt/qt_viewer.py

```diff
@@ -306,6 +307,17 @@ def __init__(self, viewer: ViewerModel, show_welcome_screen: bool = False):
         )
         self.canvas.events.draw.connect(self.camera.on_draw)
 
+        # add the FPS monitor
+        self._fps_window = 0.5
```

Comment:
I'm not sure if this should be a settings parameter, a hard coded constant, or a private property.

Reply:
I would say setting later on, but for now here it's good.